CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Release Date: _not published yet_
 - Added new opt-in `component_royalty_config` to the `/state/entity/details` endpoint. When enabled `royalty_config` will be returned for each component.
 - Use strong type definition for the `royalty_config` property of package blueprint and general components details. This is a change to OAS definition only and does not impact returned data format.
 - Introduced upper limit to the overall number of the filters used in the `/stream/transactions` endpoint, defaults to 10.
+- Introduced two new endpoints `/state/account/page/resource-preferences` and `/state/account/page/authorized-depositors` which allows to read account deposit settings.
```

Comment:
Minor notes:
1. Sounds like two endpoints should provide two functionalities but only one is listed? "account deposit settings" sounds like a higher-level concept that uses both of them, right? Guess we should stick to the more accurate, low-levelish description.
2. Having two new features I'd go with two bullet-points, i.e. describe them separately.